tests/ut/test_iterator.cc

```diff
@@ -141,9 +158,10 @@ TEST_CASE("Test Iterator Mem Index With Float Vector", "[float metrics]") {
 
     SECTION("Test Search with Bitset using iterator insufficient results") {
         using std::make_tuple;
-        auto [name, gen, threshold] = GENERATE_REF(table<std::string, std::function<knowhere::Json()>, float>({
-            make_tuple(knowhere::IndexEnum::INDEX_HNSW, hnsw_gen, hnswlib::kHnswSearchKnnBFFilterThreshold),
-        }));
+        auto [name, gen, threshold] = GENERATE_REF(table<std::string, std::function<knowhere::Json()>, float>(
+            {make_tuple(knowhere::IndexEnum::INDEX_HNSW, hnsw_gen, hnswlib::kHnswSearchKnnBFFilterThreshold),
+             make_tuple(knowhere::IndexEnum::INDEX_FAISS_IVFFLAT, ivfflat_gen, 1.0f),
+             make_tuple(knowhere::IndexEnum::INDEX_FAISS_IVFFLAT_CC, ivfflatcc_gen, 1.0f)}));
```

Comment:
Add a unit test to IVFFLAT_CC to ensure old iterators will not see new vectors.

Reply:
added.